src/main/kotlin/no/nav/familie/ef/iverksett/behandlingstatistikk/BehandlingstatistikkService.kt

```diff
@@ -0,0 +1,99 @@
+
+    @Transactional
+    fun lagreBehandlingstatistikk(behandlingstatistikk: BehandlingStatistikkDto) {
+        val behandlingDVH = mapTilBehandlingDVH(behandlingstatistikk)
+        behandlingstatistikkRepository.lagre(behandlingstatistikk.behandlingId,
+                                             behandlingDVH,
+                                             behandlingstatistikk.hendelse)
+        behandlingstatistikkProducer.sendBehandling(behandlingDVH)
+    }
+
+    private fun mapTilBehandlingDVH(behandlingstatistikk: BehandlingStatistikkDto): BehandlingDVH {
+
+        return when (behandlingstatistikk.hendelse) {
+            Hendelse.MOTTATT -> {
+                BehandlingDVH(behandlingId = behandlingstatistikk.behandlingId.toString(),
+                              sakId = behandlingstatistikk.saksnummer,
+                              personIdent = behandlingstatistikk.personIdent,
+                              registrertTid = behandlingstatistikk.hendelseTidspunkt,
+                              endretTid = behandlingstatistikk.hendelseTidspunkt,
+                              tekniskTid = ZonedDateTime.now(),
+                              behandlingStatus = Hendelse.MOTTATT.name,
+                              opprettetAv = sjekkStrengtFortrolig(behandlingstatistikk.strengtFortroligAdresse,
+                                                                  behandlingstatistikk.gjeldendeSaksbehandlerId),
+                              saksnummer = behandlingstatistikk.saksnummer,
+                              mottattTid = behandlingstatistikk.søknadstidspunkt,
+                              saksbehandler = behandlingstatistikk.gjeldendeSaksbehandlerId,
+                              opprettetEnhet = sjekkStrengtFortrolig(behandlingstatistikk.strengtFortroligAdresse,
+                                                                     behandlingstatistikk.opprettetEnhet),
+                              ansvarligEnhet = sjekkStrengtFortrolig(behandlingstatistikk.strengtFortroligAdresse,
+                                                                     behandlingstatistikk.ansvarligEnhet),
+                              behandlingMetode = "MANUELL",
+                              avsender = "NAV enslig forelder",
+                              behandlingType = behandlingstatistikk.behandlingstype.name,
+                              sakYtelse = behandlingstatistikk.stønadstype.tilKlassifisering()
+                )
+
+            }
+            Hendelse.PÅBEGYNT -> {
+                val behandlingDVH = behandlingstatistikkRepository.hent(behandlingstatistikk.behandlingId, Hendelse.MOTTATT)
+                behandlingDVH.copy(endretTid = behandlingstatistikk.hendelseTidspunkt,
+                                   tekniskTid = ZonedDateTime.now(),
+                                   saksbehandler = sjekkStrengtFortrolig(behandlingstatistikk.strengtFortroligAdresse,
```

Comment:
Skal vi sende `saksbehandler: "-5"` hvis personen er strengt fortrolig ?

Reply:
Litt lengre opp sender vi `saksbehandler = behandlingstatistikk.gjeldendeSaksbehandlerId`

Reply:
ja, riktig vi burde ha sjekkStrengtFortrolig for alle saksbehandler felter. Samme gjelder ansvarligBeslutter.